pkg/eventstore/sql_schema_adapter.go

```diff
@@ -0,0 +1,20 @@
+package eventstore
```

Comment:
I was thinking whether we'd want `eventstore/postgres` and `eventstore/<driver>` repositories. This file says we wouldn't, but `defaultInsertMarkers` could easily just be hardcoded in both `postgres` and `sqlite` if it means we have a clearer package structure.

Especially if there's more drivers later.

Reply:
which makes me think

if we were to go crazy with drivers, let's make them separate `go.mod`s
so that the main lib doesn't have unreasonable dependencies.

Matter of fact, if we separate into `eventstore/postgres` and others, that could be enough to have a `go.mod` so that the main lib doesn't have to rely on `pq`

Reply:
Having separate packages makes a lot of sense. The question is if we want to do it now.

Reply:
right. just communicating the need
might make an issue out of that

Reply:
https://github.com/ThreeDotsLabs/esja/issues/9